.github/ISSUE_TEMPLATE/documentation-issue.md

```diff
@@ -0,0 +1,11 @@
+**Is your feature request related to a problem?**
```

Comment:
Should we say "Is your documentation request related to a problem" or "Describe your documentation requirement"?